backend/entrypoint.test

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
is this file exactly the same as the classic backend?

Reply:
no make for Cython